vertx-web-api-contract/src/main/java/io/vertx/ext/web/api/RequestParameters.java

```diff
@@ -95,4 +97,13 @@
    */
   @Nullable RequestParameter body();
 
+  /**
+   * This method converts RequestParameters in an unique JsonObject with 6 fields: cookie, path, query, header, form, body<br/>
+   *
+   * cookie, path, query, header, form are JsonObject where keys are param names and values are param values, while body depends on body's shape and may not exist
+   *
+   * @return
+   */
+  @CacheReturn JsonObject toJsonObject();
```

Comment:
I think we should follow the vertx convention here to call it `toJson()` for consistency